packages/react-core/src/components/Chip/__tests__/ChipGroup.test.tsx

```diff
@@ -5,66 +5,292 @@ import userEvent from '@testing-library/user-event';
+  );
+
+  expect(screen.getByRole('group')).toHaveClass(`${styles.chipGroup} test`);
+});
+
+test('chip group has aria-label by default when categoryName is not passed', () => {
+  render(
+    <ChipGroup>
+      <Chip>1.1</Chip>
+    </ChipGroup>
+  );
+
+  expect(screen.getByRole('group')).toHaveAccessibleName('Chip group category');
+});
+
+test('chip group with custom aria-label', () => {
+  render(
+    <ChipGroup aria-label="test chip group">
+      <Chip>1.1</Chip>
+    </ChipGroup>
+  );
+
+  expect(screen.getByRole('group')).toHaveAccessibleName('test chip group');
+});
+
+test('chip group with category', () => {
+  render(
+    <ChipGroup categoryName="category">
+      <Chip>1.1</Chip>
+    </ChipGroup>
+  );
+  expect(screen.getByText('category')).toBeVisible();
+});
+
+test('chip group with category renders with class pf-m-category', () => {
+  render(
+    <ChipGroup categoryName="category">
+      <Chip>1.1</Chip>
+    </ChipGroup>
+  );
+  expect(screen.getByRole('group')).toHaveClass(`${styles.chipGroup} pf-m-category`);
+});
+
+test('chip group has aria-labelledby attribute', () => {
+  render(
+    <ChipGroup categoryName="category">
+      <Chip>1.1</Chip>
+    </ChipGroup>
+  );
+  expect(screen.getByRole('group')).toHaveAttribute('aria-labelledby', expect.stringContaining(`pf-random-id`));
+});
+
+test('chip group has aria-labelledby attribute set to ID value', () => {
+  render(
+    <ChipGroup categoryName="category" id="chip-group-id">
+      <Chip>1.1</Chip>
+    </ChipGroup>
+  );
+  expect(screen.getByRole('group')).toHaveAttribute('aria-labelledby', 'chip-group-id');
+});
+
+test('chip group expands', async () => {
+  const user = userEvent.setup();
+  const chips = ['Chip one', 'Really long chip that goes on and on', 'Chip three', 'Chip four', 'Chip five'];
+
+  render(
+    <ChipGroup aria-label="test">
+      {chips.map((currentChip) => (
+        <Chip key={currentChip}>{currentChip}</Chip>
+      ))}
+    </ChipGroup>
+  );
+
+  await user.click(screen.getByText('2 more'));
+  expect(screen.getByText('Chip five')).toBeVisible();
+});
+
+test('chip group with closable category', () => {
+  render(
+    <ChipGroup categoryName="category" isClosable>
+      <Chip>1.1</Chip>
+    </ChipGroup>
+  );
+  expect(screen.getByRole('group').lastChild).toHaveClass(styles.chipGroupClose);
+});
+
+test('chip group with closeBtnAriaLabel', () => {
+  render(
+    <ChipGroup categoryName="category">
+      <Chip closeBtnAriaLabel="close button aria label">1.1</Chip>
+    </ChipGroup>
+  );
+  expect(screen.getByLabelText('close button aria label')).toBeInTheDocument();
+});
+
+test('chip group onClick', async () => {
+  const onClose = jest.fn();
+  const user = userEvent.setup();
+
+  render(
+    <ChipGroup categoryName="category" isClosable onClick={onClose}>
+      <Chip>1.1</Chip>
+    </ChipGroup>
+  );
+
+  await user.click(screen.getByLabelText('Close chip group'));
+  expect(onClose).toHaveBeenCalled();
```

Comment:
Just to match what other callback tests are testing for (applies to the following onOverflowChipClick test as well):

```suggestion
  expect(onClose).toHaveBeenCalledTimes(1);
```